test/merchant_test.rb

```diff
@@ -24,7 +24,7 @@ def test_returns_name
   end
 
   def test_find_all_items_by_merchant_id
-    merchant = Merchant.new({:id => "1", :name => "John"}, parent) 
+    merchant = Merchant.new({:id => "1", :name => "John"}, parent)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.